packages/wallet-sdk/src/store/store.ts

```diff
@@ -23,6 +23,15 @@ export type SubAccount = {
   address: Address;
   factory?: Address;
   factoryData?: Hex;
+  config?: {
+    capabilities: Record<string, unknown>;
+  };
+};
+
+type SubAccountConfig = {
+  toOwnerAccount?: ToOwnerAccountFn;
+  capabilities?: Record<string, unknown>;
```

Comment:
same here?

Reply:
capabilities that we get from the sdk config e.g. sub account, spend limits, maybe paymaster

Reply:
makes sense. I wasn't following where they got set initially.